### PR TITLE
Ch6442/cars import traffic shifting

### PR DESCRIPTION
**Pull request by notbrain**
- Traffic shifting deployment 10% 1 min using DeploymentPreference ErrorsRollbackAlarm 
- Typescript
- eslint update and lint hooks with husky
- some reordering of code to appease linting

### CODE REVIEW

**Comment by ste00martin** [comment]
you should delete `.babelrc`

File: package.json [hunk 1]
```diff
@@ -13,16 +14,32 @@
     "request-promise": "^4.2.2"
   },
   "devDependencies": {
+    "@volta/eslint-config-volta": "^0.2.0",
     "babel-cli": "^6.24.0",
```
Comment (ste00martin) [inline package.json]: get rid of all the `babel`

**Comment by cazzer** [comment]
@notbrain sorry for leaving you high and dry on all the `babel` junk, mind just dropping all the dependencies and configs for it?

File: package.json (hunk 1 above)
Comment (notbrain) [reply package.json]: Thanks, will do.

**Pull request merged**